distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Worker.java

```diff
@@ -135,7 +135,7 @@ public void start() throws Exception {
             long workPoolCheckPeriod = v.getWorkPoolCheckPeriod();
             if (workPoolCheckPeriod != 0) {
                 workPoolReschedulableScheduler.schedule(
-                        () -> Schedule.withDelay(v.getWorkPoolCheckPeriod()),
+                        DynamicProperty.of(Schedule.withDelay(v.getWorkPoolCheckPeriod())),
```

Comment:
Prev version: when job starts to return new value from getWorkPoolCheckPeriod DJM will update schedule. 
New version: DJM will not react on such changes and will not see that  getWorkPoolCheckPeriod started to return different value. 

Suggestion :let's return DynamicProperty.of(Supplier<T>) method back to dynamic property to support such case.